sample(REPORT-368099): Custom parameter UI assignment

SathishkumarSaravanan opened the pull request:
<table>
<tr>
<td>
Description:
</td>
<td>
Custom parameter UI assignment
</td>
</tr>
<tr>
<td>
Analysis:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Reason for not identifying earlier:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Is it a breaking issue?
</td>
<td>
No
</td>
</tr>
<tr>
<td>
Solution description:
</td>
<td>
Custom parameter UI assignment
</td>
</tr>
<tr>
<td>
Areas affected and ensured:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test cases:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Tested sample location:
</td>
<td>
NA
</td>
</tr>
</table>

ssathishkumar27 committed:
sample(REPORT-368099): Custom parameter UI assignment

ManoranjanRajendran commented:
Hi @SathishkumarSaravanan 

Use the proper naming for samples.


Remove the unwanted files.
![image](https://github.com/user-attachments/assets/2720fdf9-6d26-4daf-9d1c-ca749f693d9d)


Add the sample images in PR.

SathishkumarSaravanan committed:
sample(REPORT-368099): Custom parameter UI assignment

SathishkumarSaravanan commented:
Hi @ManoranjanRajendran ,

I have addressed the review comments. Please review and merge this PR.

**SNAP:**
![image](https://github.com/user-attachments/assets/d7c620dd-8782-489a-a512-f921f84e6fc8)

ManoranjanRajendran reviewed (approved):
Changes fine
